### PR TITLE
view invoices, replace card, and new owner gets future invoices

### PR DESCRIPTION
**Pull request by EdwardAngert**
## Summary

**[Billing in the Site Dashboard](http://pantheon.io/docs/site-billing#view-a-recent-invoice-from-the-dashboard)** - Added instructions on where to find recent invoices from the Dashboard and notes explaining that they'll be visible with the next billing cycle (added in multiple docs). Also added notes to relevant docs about the timing of ownership transfers. Ownership transfers are effective immediately, and the invoices will be generated at the next billing cycle.

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6045](https://pr-6045--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `54` | `4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-6045` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/24183/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/24183/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6045--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by EdwardAngert** [comment]
taking this back to `in progress` to update sections in https://pantheon.io/docs/team-management#change-the-site-owner and https://pantheon.io/docs/change-management

File: source/partials/replace-credit-card.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The **Replace Card** button allows you to change the credit card to which a site is being billed, or to update an existing card's billing or expiration information.
+
+1. From the User Dashboard, click the **<span class="glyphicons glyphicons-cogwheel"></span> Account** tab and select **Billing** from the left side.
+
+1. In the box that appears, check the box next to **Yes, I want to continue** and click **Continue**.
```
Comment (alexfornuto) [inline source/partials/replace-credit-card.md:5]: Is "box" the right term? Is this a modal? Can we be more descriptive about which item on the page we're referring to?

Sorry, if I had a user with paid sites I'd look for myself.

**Review by alexfornuto — COMMENTED** [review]
Looks good, just have a few outstanding questions I couldn't answer myself.

File: source/content/site-owner-faq.md [hunk 2]
```diff
@@ -63,7 +63,9 @@ See the steps in our [Site Access](/site-access) doc for recovery instructions.
 
 
 ### How do I change site service levels?
-From your Site Dashboard, click **Settings**. Select a plan, and click **Update Plan**. Next, enter the payment information or invite someone to pay for the site, and click **Purchase Plan**.
+From the Site Dashboard, click the button that shows the current plan or **Upgrade** next to the site's name. Select a plan, and follow the prompts to pay or transfer site ownership to invite another user to pay.
```
[the remark this answers is not in the document]
Comment (EdwardAngert) [reply source/content/site-owner-faq.md:66]: there's a word for that element that Product uses, but I don't think would help describe it. How would you feel about `label`?

File: source/partials/replace-credit-card.md (hunk 1 above)
Comment (EdwardAngert) [reply source/partials/replace-credit-card.md:5]: it's a modal, but there's an entire step or two that vanished from here. fixing

File: source/content/site-owner-faq.md [hunk 3]
```diff
@@ -33,8 +31,8 @@ When you delete a user from a site, they lose the ability to perform any operati
 
 For more information on managing teams, see the [Team Management article](/team-management).
 
-
 ### How do I add a Supporting Agency?
```
Comment (alexfornuto) [inline source/content/site-owner-faq.md:34]: Should this get the same question -> statement treatment?

Comment (alexfornuto) [reply source/content/site-owner-faq.md:34]: Same for the subsequent H3s

File: source/content/site-plan.md [hunk 4]
```diff
@@ -11,11 +11,13 @@ Changing your site plan is typically done at launch time. For a comprehensive st
 If your site benefits from [Preferred Pricing](https://pantheon.io/plans/agency-preferred-pricing), contact your Supporting Organization for assistance, in order to retain your special pricing rate.
 
 ## Access Site Plan
+
 1. Go to the Site Dashboard.
-2. For Sandbox sites, click the **Upgrade** button:
- ![Upgrade plan button shown on Sandbox sites](../images/dashboard/upgrade-plan.png)
- Otherwise, click the **current plan**:
- ![Change current plan for paid sites](../images/dashboard/change-plan.png)
+1. For Sandbox sites, click **Upgrade** next to the site's name:
```
Comment (alexfornuto) [inline source/content/site-plan.md:16]: Please add newlines between numbered steps, Gatsby formats them better (don't ask me why).

File: source/content/site-plan.md [hunk 5]
```diff
@@ -115,14 +126,16 @@ Did you know Pantheon offers savings for sites purchased with annual billing? Se
 </Alert>
 
 ### Select Plan
+
 1. Go to the Site Dashboard.
-2. For Sandbox sites, click the **Upgrade** button. Otherwise, select the label for the site's current plan.
-3. Click **Select** to switch plans:
+1. For Sandbox sites, click **Upgrade** next to the site's name. Otherwise, click the current plan tag next to the site's name.
```
Comment (alexfornuto) [inline source/content/site-plan.md:131]: See newline comment above.

**Pull request merged**